joint_limits:
  waist:
    has_position_limits: true
    min_position: 0
    max_position: 5.06145
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: false
    max_effort: 1.0
    
  elbow:
    has_position_limits: true
    min_position: -3.1415
    max_position: 3.1415
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: false
    max_effort: 1.0

  shoulder:
    has_position_limits: true
    min_position: -3.1415
    max_position: 3.1415
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: false
    max_effort: 1.0
 
  wrist_rotation:
    has_position_limits: true
    min_position: -3.1415
    max_position: 3.1415
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: false
    max_effort: 1.0  

  wrist_flex:
    has_position_limits: true
    min_position: -1.5708
    max_position: 1.5708
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: false
    max_effort: 1.0  

  hand_roll:
    has_position_limits: true
    min_position: -3.1415
    max_position: 3.1415
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: false
    max_effort: 1.0
